[master < ] Transaction metadata (#912)
memgraph/docs: docs/reference-guide/transactions.md

## Managing transactions

Memgraph can return information about running transactions and allow you to terminate them. 

> ### Get information about transaction

Reviewer: ```suggestion
### Get information about transactions
```

> ### Get information about transaction

Reviewer: ```suggestion
### Show running transactions
```

To get information about running transaction execute the following query: 

```cypher
SHOW TRANSACTIONS;
``` 

The query shows only the transactions you started or transactions for which you
have the necessary [privileges](#privileges-needed-to-manage-all-transactions).

<img src={require('../data/how-to-guides/manage-transactional-queue/show_transactions.png').default}/>

If you are connecting to Memgraph using a client, you can pass additional
metadata when starting a transaction (if the client supports additional
metadata) which will be visible when running the `SHOW TRANSACTIONS;` query,

> thus allowing you to identify each transaction precisely.

Author: Sure! :)
I'll let you know when I push it.
Thanks for your help.

> thus allowing you to identify each transaction precisely.

Reviewer: Always and 4ever :D

> thus allowing you to identify each transaction precisely.

Author: Pushed the example. Let me know what you think.

## Managing transactions

Memgraph can return information about running transactions and allow you to terminate them.

### Show running transactions

To get information about running transaction execute the following query:

```cypher
SHOW TRANSACTIONS;
```

The query shows only the transactions you started or transactions for which you
have the necessary [privileges](#privileges-needed-to-manage-all-transactions).

<img src={require('../data/how-to-guides/manage-transactional-queue/show_transactions.png').default}/>

If you are connecting to Memgraph using a client, you can pass additional
metadata when starting a transaction (if the client supports additional
metadata) which will be visible when running the `SHOW TRANSACTIONS;` query,
thus allowing you to identify each transaction precisely.

> Below is a simple python example demonstrating how to pass metadata for
> both an implicit and explicit transaction.

Reviewer: ```suggestion
The Python example below demonstrates how to pass metadata for
both an implicit and explicit transaction:
```

<img src={require('../data/how-to-guides/manage-transactional-queue/show_transactions.png').default}/>

### Distinguish between transactions

> Since Memgraph v2.9.0, users can pass additional metadata when starting a transaction. 
> The `SHOW TRANSACTIONS;` returns the user-defined transaction metadata.
> This way, each unique transaction is easily identifiable.

Reviewer: ```suggestion
You can pass additional metadata when starting a transaction which will be visible when running the `SHOW TRANSACTIONS;` query, thus allowing you to identify each transaction.
```
